easybuild/easyconfigs/d/DendroPy/DendroPy-4.4.0-intel-2019a.eb

```diff
@@ -0,0 +1,36 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2013-2014 The Cyprus Institute
+# Authors::   Thekla Loizou <t.loizou@cyi.ac.cy>
+# License::   MIT/GPL
+# $Id$
+#
+##
+easyblock = 'PythonPackage'
+
+name = 'DendroPy'
+version = '4.4.0'
+
+homepage = 'https://pypi.python.org/pypi/DendroPy/'
+description = """A Python library for phylogenetics and phylogenetic computing: 
+reading, writing, simulation, processing and manipulation of phylogenetic trees 
+(phylogenies) and characters."""
+
+toolchain = {'name': 'intel', 'version': '2019a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['f0a0e2ce78b3ed213d6c1791332d57778b7f63d602430c1548a5d822acf2799c']
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+download_dep_fail = True
+use_pip = True
```

Comment:
@boegel don't you have to add `check_ldshared = True`?

Reply:
No, because this a pure Python package (no compilation of C/C++ code or anything like that, only `*.py` files)